obal/data/modules/srpm.py

```diff
@@ -47,6 +47,9 @@ def copy_sources(spec_file, package_dir, sources_dir):
         if not source:
             continue
 
+        if not 'Source' in source and not 'Patch' in source:
```

Comment:
I find this easier to read:
```suggestion
        if 'Source' not in source and 'Patch' not in source:
```

Looking at the output, this is also more accurate:
```suggestion
        if not (source.startswith('Source') or source.startswith('Patch')):
```